lib/money/core_extensions.rb

```diff
@@ -12,7 +12,7 @@ def to_money
 #   '100'.to_money => #<Money @cents=10000>
 #   '100.37'.to_money => #<Money @cents=10037>
 class String
-  def to_money
-    empty? ? Money.empty : Money.parse(self)
+  def to_money(currency = Money.default_currency)
```

Comment:
This doesn't seem consistent with `Money.new`'s currency defaulting to `nil`. Either this should be match `Money.new` or make it match this format. Alternatively you could splat the args and pass any number of args to `Money.new`